content/who-we-are/team/_index.en.md

```diff
@@ -9,48 +9,151 @@ intro: >-
   been in the field for a long time.
 
 
-  Not every member has a profile picture or is listed in a team. Privacy matters!
+  Not every member has a profile picture or is listed in a team. Privacy
+  matters!
 in_progress: false
 teams:
   - title: Board
-    description: At the head of DIVD is our board, providing guidance, direction,
-      and making strategic decisions. The day-to-day operations are managed by a
+    description: >-
+      At the head of DIVD is our board, providing guidance, direction, and
+      making strategic decisions. The day-to-day operations are managed by a
       team that includes our director, department heads, the CISO, and the
       crisis manager.
-    members: []
+    members:
+      - /who-we-are/team/people/eleonora-petridou
+      - /who-we-are/team/people/inge-bryan
+      - /who-we-are/team/people/joost-hendricksen
+      - /who-we-are/team/people/marinus-kuivenhoven
+      - /who-we-are/team/people/shairesh-algoe
```

Comment:
Hier gaan we nog wat op verzinnen, maar voor nu is het het makkelijkste als je deze file niet mee commit :)

Reply:
Good call 😓 daarvoor hebben we dus reviews :-p

Reply:
Is verwijderd